ppx_cstruct.opam

```diff
@@ -23,5 +23,6 @@ depends: [
   "ocaml-migrate-parsetree"
   "ppx_driver"    {test & >= "v0.9.0"}
   "ppx_sexp_conv" {test}
+  "cstruct-unix"  {test}
```

Comment:
I think there are different default for the "classic" TravisCI scripts vs. the Docker-based ones. The classic ones have TEST=true by default.